modules.d/95iscsi/iscsiroot.sh

```diff
@@ -238,14 +233,22 @@ handle_netroot()
                      ${iscsi_in_username:+--name=node.session.auth.username_in --value=$iscsi_in_username} \
                      ${iscsi_in_password:+--name=node.session.auth.password_in --value=$iscsi_in_password} \
                      $EXTRA \
-                     $NULL
-        ;;
-        *)
-        ;;
-        esac
+                     $NULL"
+            $CMD
+            if [ "$netif" != "timeout" ]; then
+                $CMD --login
```

Comment:
```
[   11.121564] dracut-initqueue[571]: iscsiadm: Invalid parameters. Login/logout and op passed in
```

Reply:
Fixed with the update 403256a (pushed Oct 1st).